_posts/2021-03-29-Activism Poster.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+author: Amna Amin
+title: Activism Poster 
+---
+
+## Explanation of Thought Process 
+
+The document that I am coding is an image used to advertise an event from Muslim Student Activists. This document is interesting and potentially useful to historians that want to note what topics student activists focused on and how they advertised events. Thus, the aspects of the document that I want to preserve for researchers is noting what titles are used, how information is presented (in forms of questions or demands for instance), where and when these events take place, and images used on the poster. 
+
+Based on the elements I want to document, I want to note the relative size and importance of different elements of the text. For instance, the title can be noted using markup codes like <head>. Additionally, I want to note the group’s hosting so code that includes elements like <orgName> were good markup codes to use. Overall, I also used tags like question to indicate that questions were being asked, image for an image being shown, date to show a date was given for the event, and location to indicate the physical space the event was held in. 
```

Comment:
In order for these tags to appear as inline text, you'll need to set them off with single tick marks. Otherwise, anything within brackets is interpreted as code. So you need to wrap `<head>` and `<orgName>` in single tick marks (I did that here so that they show up in my comments).